components/ScheduleSection.js

```diff
@@ -0,0 +1,240 @@
+            speakerTitle='(Front-end Engineer at Jitta)'
+          />
+        </SessionRow>
+        <BreakRow time='Break' />
+        <SessionRow time='15:15 - 16:00'>
+          <SessionItem
+            topic='Angualar to React without breakingthings'
+            speakerName='Jirat Kijlerdpornpailoj'
+            speakerTitle='(SPRINT3r / Gucode)'
+          />
+          <SessionItem
+            topic='Testable React: Patterns and comparison'
+            speakerName='Chakrit Likitkhajorn'
+            speakerTitle='(Lead Developer at Taskworld)'
+          />
+        </SessionRow>
+        <SessionRow time='16:00 - 16:45'>
+          <SessionItem
+            topic='Workshop with Next.js'
+            speakerName='Nati Namvong'
+            speakerTitle='(CTO at Sellsuki)'
+          />
+          <SessionItem
+            topic='Lessons Learned from using Next.js in Production'
+            speakerName='Panjamapong Sermsawatsri'
+            speakerTitle='(CTO at TakeMeTour)'
+          />
+        </SessionRow>
+      </div>
+      <style jsx>{`
+        .schedule {
+          text-align: center;
+        }
+        .container {
+          max-width: 1060px;
+          width: 100%;
+          margin: 0 auto;
+          text-align: left;
+        }
+        @media (max-width: 768px){
+          .container {
+            max-width: 750px;
+            text-align: center;
+          }
+        }
+      `}</style>
+    </div>
+  )
+}
+
+const getSpeakerAvatar = (name) => {
+  return `static/speakers/${name.replace(' ', '')}.jpg`
+}
+
+const ScheduleRow = ({ children }) => (
+  <div className='row _margin-bottom-30'>
+    {children}
+    <style jsx>{`
+      .row {
+        position: relative;
+        display: flex;
+        flex-wrap: wrap;
+        padding-left:7.5px;
+        padding-right:7.5px;
+      }
+      ._margin-bottom-30 {
+        margin-bottom: 30px;
+      }
+    `}</style>
+  </div>
+)
+
+const ScheduleColumn = ({ children, size }) => {
+  return (
+    <div className='column' data-size={size}>
+      {children}
+      <style jsx>{`
+        .column { width: 100%; }
+        @media (min-width: 768px) {
+          .column[data-size="2"] { width: 16.66%; }
+          .column[data-size="5"] { width: 41.65%; }
+          .column[data-size="8"] { width: 66.64%; }
+          .column[data-size="10"] { width: 83.33%; }
+        }
+      `}</style>
+    </div>
+  )
+}
+
+const ActivityRow = ({ time, children }) => (
+  <ScheduleRow>
+    <ScheduleColumn size={2}>
+      <SessionItemTime>{time}</SessionItemTime>
+    </ScheduleColumn>
+    <ScheduleColumn size={10}>
+      <div className='main'>{children}</div>
+    </ScheduleColumn>
+    <style jsx>{`
+      .main {
+        font-size:28px;
+      }
+    `}</style>
+  </ScheduleRow>
+)
+
+const BreakRow = ({ time }) => (
+  <ScheduleRow>
+    <ScheduleColumn size={2}>
+      <SessionItemTime>{time}</SessionItemTime>
+    </ScheduleColumn>
+    <ScheduleColumn size={8}>
```

Comment:
The break separator is not center vertically-aligned.

![image](https://cloud.githubusercontent.com/assets/193136/26300306/e8a78de4-3f07-11e7-9745-f6a4514388ca.png)

This can be fixed by `<ScheduleColumn size={8} alignSelf='center'>`